mango-core/src/main/scala/org/bdgenomics/mango/filters/GenericFilter.scala

```diff
@@ -37,4 +45,16 @@ object GenericFilter {
     }
   }
 
+  /**
+   * Reformat regions to fit window size
```

Comment:
Can you put a description of why this is necessary (i.e. this is for grouping counts to be displayed in the frontend or something)